Project/MyGameLibrary/Character/Character.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+
+namespace Fall2020_CSC403_Project.code
+{
+    public class Character
+    {
+        public string CharacterName { get; set; };
+        public bool isDead = false;
+        private int LoveScore;
+        public int GetLoveScore() {
+            return LoveScore;
+        }
+        public void SetLoveScore(int LScore) {
+            if (isDead == true)
+            {
+                LoveScore = -1;
+            }
+            else if (LoveScore > 100)
+            {
+                LoveScore = 100;
+            } else if (LoveScore < 0)
+            {
+                LoveScore = 0;
+            }
+            else {
+                LoveScore = LScore;
+            }
+        }
+
+        //Items is ID of item, int is love point value of gift for this character
+        public Dictionary<Items, int> ItemScores; 
+
+        public Character(string name, int LScore, Dictionary<Items, int> dict)
+        {
+            CharacterName = name;
+            LoveScore = LScore;
+            ItemScores = dict;
+        }
+
+        public void LoveUpdate(Items item)
+        {
+            SetLoveScore(LoveScore+ItemScores[item]);
```

Comment:
When you update it to the C# built in getters and setters functionality you own't need this line you're just be able to add to the property to change it.